src/core/grpc_service.proto

```diff
@@ -370,47 +370,69 @@ message InferSharedMemoryRegion
 //@@
 message SharedMemoryControlRequest
 {
-  //@@  .. cpp:enum:: Type
+  //@@  .. cpp:var:: message RegisterInfo
   //@@
-  //@@     Types of control operations for shared memory
+  //@@     The information needed to register the shared memory region.
   //@@
-  enum Type {
-    //@@    .. cpp:enumerator:: Type::REGISTER = 0
+  message RegisterInfo
```

Comment:
Drop "info" here and elsewhere. Just "Register".

Reply:
Sure